File: Dot net Backend/latest/Fleet_Management/DTO/BookingDetailDTO.cs
using System;

namespace Fleet_Management.DTO
{
    public class BookingDetailDTO
    {
        public long BookingDetailId { get; set; }
        public long AddonId { get; set; }
        public double AddonRate { get; set; }

        public BookingDetailDTO() { }

        public BookingDetailDTO(long bookingDetailId, long addonId, double addonRate)
        {
            BookingDetailId = bookingDetailId;
            AddonId = addonId;
            AddonRate = addonRate;
        }
    }
}

File: Backend/demo/src/main/java/com/example/dto/BookingDetailDTO.java
package com.example.dto;

public class BookingDetailDTO {
    private Long bookingDetailId;
    private Long addonId;
    private Double addonRate;

    public BookingDetailDTO(Long bookingDetailId, Long addonId, Double addonRate) {
        this.bookingDetailId = bookingDetailId;
        this.addonId = addonId;
        this.addonRate = addonRate;
    }

	public Long getBookingDetailId() {
		return bookingDetailId;
	}

	public void setBookingDetailId(Long bookingDetailId) {
		this.bookingDetailId = bookingDetailId;
	}

	public Long getAddonId() {
		return addonId;
	}

	public void setAddonId(Long addonId) {
		this.addonId = addonId;
	}

	public Double getAddonRate() {
		return addonRate;
	}

	public void setAddonRate(Double addonRate) {
		this.addonRate = addonRate;
	}


}